cyral/resource_cyral_policy_rule_test.go

```diff
@@ -118,16 +118,19 @@ func formatPolicyRuleConfigIntoConfig(data PolicyRuleConfig) string {
 			data = ["TEST_CCN"]
 			rows = 1
 			severity = "%s"
+			rate_limit = 1
```

Comment:
Could you please add some test checks to this test? Looks like its currently only checking the `severity` field

Reply:
Ok, I will update this with this values, I thought that only applied inside the provider was ok, but make sense do like `severity`